cmake/Modules/FindMKL.cmake

```diff
@@ -168,6 +168,25 @@ IF (EXISTS ${INTEL_OMP_DIR})
   ENDIF()
 ENDIF()
 
+MACRO(GET_MKL_LIB_NAMES LIBRARIES INTERFACE MKL64)
+    cmake_parse_arguments("" "" "USE_STATIC;THREAD" "" ${ARGN})
+    set(${LIBRARIES} mkl_${INTERFACE}${MKL64}
+        mkl_core
+       )
+    if(_THREAD)
+        LIST(INSERT ${LIBRARIES} 1 ${_THREAD})
+        if(_USE_STATIC)
```

Comment:
Where `_USE_STATIC` is defined?

Reply:
Its an argument to the macro - the caller of the macro passes it in

Reply:
I have updated the macro to just use the `USE_STATIC_MKL ` option (which is global) instead of making the caller pass it in.